src/components/pages/GamePlay/GameWriteStep.js

```diff
@@ -45,11 +52,15 @@ export default function GameWriteStep(props) {
         buttonTxt: "Let's Go!",
       };
 
-      props.enableModalWindow(modalData);
+      //props.enableModalWindow(modalData);
 
       setIsUploading(false);
     }, triggerSubmitTimer);
-    history.push('/child/next-steps');
+    if (props.submissionData.HasDrawn) {
```

Comment:
great job implementing this check, it'll be crucial as we move toward creating submission records.